rust/src/graph/graph_manipulation.rs

```diff
@@ -118,7 +118,7 @@ impl Graph {
         &mut self,
         importer: ModuleToken,
         imported: ModuleToken,
-        line_number: usize,
+        line_number: u16,
```

Comment:
This gives a small but measurable performance benefit. I think people aren't really having import statements past line 65535 😅

Reply:
I agree with the change - but I can believe there are modules out there in the wild that get that big. How would Grimp behave if it did encounter such a module?

Ideally we should handle it gracefully, but as long as we have the new behaviour captured in a ticket, and the change mentioned in the CHANGELOG, I'm happy.

Reply:
> How would Grimp behave if it did encounter such a module?

Yes maybe it's worth a bit of thought.

It's quite a micro-optimisation anyways, so I think instead I'll just opt to use u32 here instead - I'm near certain that nobody will overflow that!

Reply:
(With u32 I wouldn't update the changelog)

Reply:
> I'm near certain that nobody will overflow that!

Agreed 😄